src/darwin/Framework/CHIPTests/CHIPSetupPayloadParserTests.m

```diff
@@ -171,7 +171,7 @@ - (void)testManualParser
     XCTAssertEqual(payload.setUpPINCode.unsignedIntegerValue, 123456780);
     XCTAssertEqual(payload.vendorID.unsignedIntegerValue, 1);
     XCTAssertEqual(payload.productID.unsignedIntegerValue, 1);
-    XCTAssertTrue(payload.requiresCustomFlow);
+    XCTAssertEqual(payload.commissioningFlow, kCommissioningFlowCustom);
```

Comment:
And here, why the change?

Reply:
The boolean changed to enum, so I had to change it. Or am I missing something?

Reply:
No, this was me being confused, I think.